manager/integration/tests/test_upgrade.py

```diff
@@ -31,7 +48,7 @@ def longhorn_upgrade(image_tag):
 
 
 @pytest.mark.upgrade
-def test_upgrade(upgrade_image_tag, volume_name):  # NOQA
+def test_upgrade(upgrade_image_tag, volume_name, pod_make, statefulset, storage_class):  # NOQAA
     """
     Test Longhorn upgrade
 
```

Comment:
Add prerequisite. 

As I mentioned, `auto-salvage` should be disabled here.

Reply:
Also, update the skeleton.

Reply:
Updated